src/main/resources/intentionDescriptions/UnwrapToTryIntention/description.html

```diff
@@ -1,5 +1,6 @@
 <html>
 <body>
-This intention transforms an expression with unwrap to a try operator.
+This intention transforms an expression with <code>.unwrap()</code> to a
```

Comment:
Actually, I think `.` is redundant here and in all similar cases. I'd write `<code>unwrap()</code> method call` instead

Reply:
I think the dot syntax looks better. It highlights that the function is a method call. Perhaps we should add some placeholder receiver to avoid a dangling dot?